app/components/layout/SideNav.tsx

```diff
@@ -12,7 +12,7 @@ export function SideNav() {
       {navItems.map((item) => {
         return (
           <Link
-            to={item.url}
+            to={`/${item.url}`}
```

Comment:
Yeah I noticed these relative URL's have bene causing issues... Wonder what's the root cause, maybe Remix bug?